### PR TITLE
Update efficientnet_lite.py 

### PR DESCRIPTION
**Pull request by vijaybirju**
File cleaned remove tf.keras from keras

### CODE REVIEW

File: keras_cv/models/efficientnet_lite.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 import math
 
 import tensorflow as tf
+from tf import Keras
```
Comment (innat) [inline keras_cv/models/efficientnet_lite.py:27]: from tensorflow and not tf

File: keras_cv/models/efficientnet_lite.py [hunk 2]
```diff
@@ -24,6 +24,7 @@
...
 from keras import backend
 from keras import layers
```
Comment (innat) [inline keras_cv/models/efficientnet_lite.py:29]: import from tensorflow

File: keras_cv/models/efficientnet_lite.py (hunk 1 above)
Comment (vijaybirju) [reply keras_cv/models/efficientnet_lite.py:27]: ok

File: keras_cv/models/efficientnet_lite.py (hunk 2 above)
Comment (vijaybirju) [reply keras_cv/models/efficientnet_lite.py:29]: thank you made it

**Comment by ianstenbit** [comment]
Hi @vijaybirju -- this was fixed in #1530 which was just merged.

Sorry for the overlap here, and thank you for your PR!

**Pull request closed without merging**